src/admin/dashboard.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Box, Header } from 'admin-bro';
+
+const Dashboard = () => {
+  return (
+    <Box variant="grey">
```

Comment:
_Box_ is a component built-in with the admin, but this is essentially a React component..., so we should be able to do any desired customization on a real project.